python/runLumiPixel0SimBoxRestGas.C

```diff
@@ -0,0 +1,204 @@
+// Panda FullSim macro
```

Comment:
I would prefer to add this to PandaROOT instead, all the other run macros are there too.